tools/haskell-language-server/default.nix

```diff
@@ -0,0 +1,9 @@
+{ sources }:
+
+ghcVersion: import sources.nix-hls {
+  inherit ghcVersion;
+  sources = {
```

Comment:
This sources list is not as complete as Sukant's, but some of his sources are used in his support module, which looks lik a niv alternaive.  This is the minimal set that lets us string our versions through to his derivation.

This could be fragile as we update, but since things are pinned, we can always inline if and when this does break.